### PR TITLE
Add visualization which can read uploaded info

### PR DESCRIPTION
**Pull request by josephsnyder**
Add a new visualization  to read in the JSON information from the
FileManGlobalDataParser output and display statistics on the inbound
information.  It should contrain the modal window position and height to
stop it from moving out of view.

### CODE REVIEW

File: Visual/Documentation/QueryVis_formatting.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.. title: "Classify VistA Data formatting"
+
+============================================
+Creating the JSON for "Classify VistA Data"
+============================================
+
+The "Classify VistA Data" page accepts uploads of JSON information and attempts
```
Comment (BetsyMcPhail) [inline Visual/Documentation/QueryVis_formatting.rst]: missing 2nd half of sentence

File: Visual/queryVis_stats.php [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    <div id="header1" class="header"></div>
+    <div id="TotalCount" ></div>
+    <div id="percentage" ></div>
+    <div id="list"></div>
+    <div class="tooltipTail"></div>
+  </div>
+
+<div class="hint" style="position:relative;">
+  <p>
+    This page will accept an uploaded JSON object and attempt to show the data in one of two forms: a pie chart and a DataTables display
```
Comment (BetsyMcPhail) [inline Visual/queryVis_stats.php]: uploaded --> user-generated or standard ViViaN ?
JSON object --> JSON file ? 
DataTables display --> table ?

File: Visual/Documentation/QueryVis_formatting.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Generating from the OSEHRA Testing Repository
+---------------------------------------------
+
+The JSON information that the page expects to be uploaded is generated by the
```
Comment (BetsyMcPhail) [inline Visual/Documentation/QueryVis_formatting.rst]: Mention that this data is automatically available on the page - just need to select from drop-down

File: Visual/Documentation/QueryVis_formatting.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+FileManGlobalDataParser script. See the `Generate ViViaN and Dox`_ for
+instructions on how to execute that task.
+
+Generating from another source
+------------------------------
+
+To generate information from a different source in the expected form for the
+ViViaN page to ingest, there are a few rules that should be adhered to beyond
+the usual JSON rules (no duplication of keys, etc.)  These rules are.
```
Comment (BetsyMcPhail) [inline Visual/Documentation/QueryVis_formatting.rst]: Is there an easy link to JSON format?

File: Visual/Documentation/QueryVis_formatting.rst [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+* Each piece of data within each entry must maintain a string as an identifier
+  followed by a ":"
+* Data with multiple entries must continue to use that text value as labels for
+  entries within the data.
+
```
Comment (BetsyMcPhail) [inline Visual/Documentation/QueryVis_formatting.rst]: Use identifier or label consistently. I'm not sure if one is more standard when talking about JSON files?

**Comment by BetsyMcPhail** [comment]
Also, the navigation button isn't highlighted when the page is selected?

**Comment by josephsnyder** [comment]
Hi @BetsyMcPhail, 

When you've got the time, I think this is ready for another look.  Thanks!

File: Visual/queryVis_stats.php [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+    <div id="percentage" ></div>
+    <div id="list"></div>
+    <div class="tooltipTail"></div>
+  </div>
+
+<div class="hint" style="position:relative;">
+  <p>
+    This page will accept an JSON file and attempt to show the data in one of two forms: a pie chart and a filterable table.
+    to allow searching and filtering of the uploaded information.
+    For more information on the expected form of the uploaded JSON, see the following <a href="./Documentation/QueryVis_formatting.rst">file</a>
```
Comment (BetsyMcPhail) [inline Visual/queryVis_stats.php]: How about something like "This page will accept a JSON file and display the data in one of two forms: a pie chart or a table. For more information on the expected form of the uploaded JSON, see the following file."

**Comment by BetsyMcPhail** [comment]
Looks really good!

**Pull request merged**